### ISSUE TITLE
Discourse diacriticals widget

### ISSUE THREAD

**Issue by sujato** [suttacentral/legacy-suttacentral#118]
The widget for adding diacriticals has been brokend for some time on a Discourse upgrade, it should be fixed.

**Comment by ghost** [comment]
What is this? You can add diacritical texts just fine but that is not what you mean ... was there some kind of "character map" widget somewhere?

**Comment by sujato** [comment]
Yes, Blake made a widget on the text editor for inserting diacriticals, but it broke with an upgrade. Actually we should upgrade to the latest Discourse first, then check this. They are testing a new text editor, I think it will be in the current edition, but not sure about that.

**Comment by ghost** [comment]
@Blake,
Can you run the upgrade by running the following on the server:

cd /var/discourse
git pull
./launcher rebuild app

**Comment by blake-sc** [comment]
Okay underway. I'll also do an apt-get upgrade and reboot the server.

**Comment by ghost** [comment]
Found the plugin enable-pali-input.js.es6 but am a bit unconfident about how to go about it ... I can find that suttacentral.js is a register_asset in plugin.rb but cannot find where enable-pali-input.js.es6 is called.
I think I also need access to the server and probably some knowledge of rails.

**Comment by blake-sc** [comment]
I'll fix the Discourse plugin - it's not that much fun working with rails and ember but I've figured out enough of it to get things done without too much trouble.

**Comment by sujato** [comment]
@blake-sc 

Bump! Still brokeded …

**Comment by sujato** [comment]
Actually, debump. The composer toolbar is being rewritten for Discourse 1.5 so best wait.

https://meta.discourse.org/t/discourse-version-1-5/30609

**Comment by sujato** [comment]
Note; this is still under construction at Discourse, they are rebuilding for Common Mark. No shipping date AFAIK.
